cmd/transcriber/config/config.go

```diff
@@ -95,7 +95,9 @@ func (a TranscribeAPI) IsValid() bool {
 	}
 }
 
-func (cfg CallTranscriberConfig) IsValid() error {
+// IsValid checks the config's validity. Set inContainer to true only if this is being called
+// from the job container itself.
+func (cfg CallTranscriberConfig) IsValid(inContainer bool) error {
```

Comment:
Not sure how I feel about this. We are breaking a pretty common interface and exposing an implementation details. Have we considered some other options?

1. (naive) On the plugin side we could validate before applying any of the threads values. It will pick up defaults and pass.
2. We could use a build flag injected at compile time. If the validation is running from the `call-transcriber` binary then it will pick it up and validate any system dependent logic.

Reply:
I agree, I did #2 (probably the better way).